timeline/views_ajax.py

```diff
@@ -45,6 +45,13 @@ def get_event_details(self, event):
                     'description': s.description,
                     'type': s.status_type,
                     'class': get_status_style(s),
+                    'extra_status_link': '/timeline/ajax/extra/{}/{}'.format(
+                        event.sodar_uuid
+                        if event.project
+                        else f'site/{event.sodar_uuid}',
+                        idx,
+                    ),
+                    'no_extra_status': s.extra_data == {},
```

Comment:
The point here is that status extra data can be shown for any status (because with empty extra_data will be formed just {}). 
So to check this in backend side we have to make additional requests. 
To check this in jquery we have to make kind of string matching, what is possible, but in my opinoin not optimal way. 

So now is a following question: what exactly should I implement?

Reply:
I'm not sure if I follow. If the extra data is `{}`, we simply don't show the link. So IMHO it's sufficient to only have the field for the URL and set it empty of the extra data field is empty.

Reply:
Oh way, I overcomplicated this task in my head) Everything should be alright now